tests/logictest/suites/base/05_ddl/05_0016_ddl_stage

```diff
@@ -13,15 +13,18 @@ CREATE STAGE test_stage_internal file_format=(type=csv compression=AUTO record_d
 statement ok
 LIST @test_stage_internal;
 
-statement ok
+statement query TTTTTITT
 desc stage test_stage_internal;
 
-statement query TTITT
+----
+test_stage_internal Internal StageParams { storage: Fs(StorageFsConfig { root: "_data" }) } CopyOptions { on_error: None, size_limit: 0 } FileFormatOptions { format: Csv, skip_header: 0, field_delimiter: "", record_delimiter: "NONE", compression: Auto } 0 'root'@'127.0.0.1' 
+
+statement query TTTTT
 SHOW STAGES;
 
 ----
-test_stage External NULL 'root'@'127.0.0.1'
-test_stage_internal Internal 0 'root'@'127.0.0.1'
+test_stage External NULL 'root'@'127.0.0.1' 
```

Comment:
Is the white space expected in the end?

Reply:
> Is the white space expected in the end?

Yes, actually there are 5 columns, the last one is empty.